package.json

```diff
@@ -25,9 +30,21 @@
     "eyes.selenium": "0.0.81"
   },
   "devDependencies": {
-    "uuid": "^2.0.2",
-    "mocha": "^5.2.0",
+    "protractor": "^5.4.1",
     "selenium-webdriver": "^2.53.0",
-    "wnpm-ci": "*"
+    "uuid": "^2.0.2",
+    "wnpm-ci": "*",
+    "yoshi": "^3.7.0"
```

Comment:
maybe upgrade to latest `3.11.0`?

Reply:
It has a `^` so it takes the latest anyways.